### PR TITLE
New package: Audiveris.Audiveris version 5.3.1

### LINKED ISSUE #156109: [Package Request]: Audiveris
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/Audiveris/audiveris/releases
Publisher: [Hervé Bitteur](https://en.wikipedia.org/wiki/Audiveris)
Package Name: Audiveris
Description: `Latest generation of Audiveris OMR (optical music recognition) engine`
Package Version: **5.3.1** _(stable version)_ `AND` **5.4-alpha** _(prerelase alpha version)_
Installer URL:

[STABLE](https://github.com/Audiveris/audiveris/releases/download/5.3.1/Audiveris_Setup-5.3.1-windows-x86_64.exe)

[PRERELEASE ALPHA](https://github.com/Audiveris/audiveris/releases/download/5.4-alpha/Audiveris_Setup-5.4-alpha-windows-x86_64.exe)

### PR DESCRIPTION
**Pull request by Exorcism0666**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?
- Resolve #156109

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/156131)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 882.)

File: manifests/a/Audiveris/Audiveris/5.3.1/Audiveris.Audiveris.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Audiveris.Audiveris
+PackageVersion: 5.3.1
+PackageLocale: en-US
+Publisher: Audiveris
```
Comment (Trenly) [inline manifests/a/Audiveris/Audiveris/5.3.1/Audiveris.Audiveris.locale.en-US.yaml:7]:
```suggestion
Publisher: Audiveris Team
```
This should match control panel

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
> Is there an installer switch to bypass this message?

It doesn't look like there is
https://github.com/Audiveris/audiveris/blob/2dedb99bc16b521e9e9c85e6be6ae8a9bb22c578/dev/installer-windows/Installer.nsi#L96

[Policy] Interactive Only

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 884.)

**Comment by stephengillie** [comment]
> Automatic Validation ended with: Edit: image
> 
> Hi @Exorcism0666,
> 
> Is there an installer switch to bypass this message?

This is unchanged.

**Pull request closed without merging**